src/site/content/en/learn/pwa/assets-and-data/index.md

```diff
@@ -1,11 +1,140 @@
+* APIs and web services: API calls can be cached; or, instead of caching the API responses, you can store their data in IndexedDB.  
+
+{% Aside 'gotchas' %}
+With platform-specific apps, the most common pattern is to include all the app's assets in its package; that's why it's common these apps are larger than 500MB. On the other hand, A PWA can initially install only the minimum set of resources to offer an offline and fast experience and download the rest later, with PWAs using considerably less storage on installation. 
+{% endAside %}
+
+### Updating assets
+In the standard app model for app catalog installed apps, when developers need to update their app, they publish a new package. Users need to download the whole package again, even if most of the assets haven't changed.
+With PWAs, using the approaches on the section above, you decide how and when to update assets. 
+Here are different options for how to update assets:
+
+* Full Update: in this case, every time you make a change in the app, even a minor one, your code will download all the assets again in the cache.
+* Partial Update: in this approach, you create a method to define which assets have been updated, and you update only those, with or without the user's intervention.
+* Continuous Update: using this technique, your assets will be checked and updated automatically on every PWA usage individually
+
+{% Aside %}
+When caching assets, you need to decide on a cache swap or clean up policy that defines when the user will see an updated asset; taking into account storage usage and the balance between a fast experience and displaying a slightly stale asset.
+{% endAside %}
+
+## Size and Lifespan
+
+Usually platform-specific apps don't deal with size limits; at installation, apps can be gigabytes or more in size, as long as the device has the capacity the installation will be allowed. Also, while the app is installed, it will use that total amount of storage no matter if you use the app or not.
+Storage is handled differently for PWAs. The browser will store your assets based on policies that you define in your PWA's logic. 
+
+{% Aside %}
+The users are always in control, and they can delete the assets at any time. For PWAs, that manual eviction happens via the browser's settings. 
+{% endAside %}
+
+In terms of size limits, it depends on the browser. 
+It's not as flexible as for platform-specific apps, but it's typically good enough for most web apps. You can find the specific limitations by browser in [this article](/storage-for-the-web/#how-much).
+
+Assets can be evicted based on storage pressure, or after some weeks of inactivity, if the user is using your PWA in the browser. On some platforms, if the user installs your PWA, eviction won't happen. Where available, our code can request persistent storage through an API to avoid that eviction.
```

Comment:
```suggestion
Browsers can evict assets based on storage pressure, or after some weeks of inactivity, if the user is using your PWA in the browser. On some platforms, if the user installs your PWA, eviction won't happen. Where available, code can request persistent storage through an API to avoid that eviction.
```